aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/util/writer/CqDialogWriter.java

```diff
@@ -74,7 +74,14 @@ void populateDomDocument(Class<?> componentClass, Element root) {
         PluginRuntime.context().getXmlUtility().mapProperties(root, dialog, XmlScope.CQ_DIALOG);
         root.setAttribute(JcrResourceConstants.SLING_RESOURCE_TYPE_PROPERTY, ResourceTypes.DIALOG);
 
-        DialogLayout dialogLayout = ArrayUtils.isEmpty(dialog.tabs()) ? dialog.layout() : DialogLayout.TABS;
+        DialogLayout dialogLayout;
+        if (!ArrayUtils.isEmpty(dialog.accordionTabs())) {
```

Comment:
(cosmetic) I'd move tabs before accordion so that they have precedence in detection. Not a great deal.. just an intuition that tabs matter somewhat more

Reply:
fixed, thanks